docs/publishing-announcements.md

```
@@ -0,0 +1,189 @@
+looking at it disappears from the screen, which reads as a bug in the channel
+when it is the channel working (§3.4).
+
+## 3. Validate
+
+```sh
+dart run tool/announce.dart draft.json --out event.json
+```
+
+Either it prints
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)